src/System.Private.ServiceModel/tests/Common/Infrastructure/src/project.json

```diff
@@ -6,8 +6,8 @@
     "System.Net.Http": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-23921",
-    "System.ServiceModel.Duplex": "4.0.1-rc3-23921",
-    "System.ServiceModel.Http": "4.0.11-rc3-23921",
+    "System.ServiceModel.Duplex": "4.1.0-rc3-23921",
```

Comment:
Searching for System.Net.Http.WinHttpHandler I can only find an old version in lock.json files under C:\OSS\wcf\Tools test-runtime and tool-runtime, not sure if that is the source?

The exact same version number found there "4.0.0-rc2-23816" is also in the packages that were pulled down.

I have to leave for the day, I pushed my changes made on top of Eric's to my repo.
https://github.com/StephenBonikowsky/wcf/tree/ericstj-update23918